resources/EPvalidator.rb

```diff
@@ -127,8 +128,8 @@ def self.run_validator(hpxml_doc)
       },
 
       ## [VentedAttic]
-      "/HPXML/Building/BuildingDetails/Enclosure/Roofs/Roof[InteriorAdjacentTo='attic - vented']" => {
-        "../../Attics/Attic[AtticType/Attic[Vented='true']]/VentilationRate[[UnitofMeasure='SLA']/Value | extension/ConstantACHnatural]" => zero_or_one,
+      "/HPXML/Building/BuildingDetails/Enclosure/Roofs/Roof[InteriorAdjacentTo='attic - vented']/../../Attics/Attic[AtticType/Attic[Vented='true']]" => {
+        "VentilationRate[UnitofMeasure='SLA']/Value | extension/ConstantACHnatural" => zero_or_one,
```

Comment:
There's a subtle difference here too. Previously, if there is a roof adjacent to vented attic, we checked for a ventilation rate on an Attic element (of vented attic). With your change, the check only occurs if the Attic element exists too. So if an HPXML file has the roof element but no corresponding Attic element, the attic ventilation check no longer occurs.

I think the reference to the Attic needs to remain where it was and the xpath bugfix needs to be applied differently.

Reply:
I think the syntax is functionally the same, if there's no attic element, the attic ventilation won't be checked in either case because attic ventilation is under `Attic` element. My understanding is that the difference is, my change is stopped at searching parent element, but the previous code is stopped at searching sub element. If this hash has checks other than attic elements, it does have difference, as we want to execute those checks even when attic doesn't exist, but not in this case. 

The reason why I split these lines into one-layer relationship is because I want the format of "parent element" => {"sub element" => count} be only checking specific parent element structure, preventing checking the whole tree from root or from much higher level (as we know, it will cause issues when multiple tree branches are specified). It also looks more clear. If we still want it to be executed one by one from roof -> attic -> attic ventilation, it might require more hashes like: 

    "/HPXML/Building/BuildingDetails/Enclosure/Roofs/Roof[InteriorAdjacentTo='attic - vented']" => {
        "../../Attics/Attic[AtticType/Attic[Vented='true']] => zero_or_more,
    },

    "/HPXML/Building/BuildingDetails/Enclosure/Attics/Attic[AtticType/Attic[Vented='true']]" => {
    "VentilationRate[UnitofMeasure='SLA']/Value | extension/ConstantACHnatural" => zero_or_one, 
    },

Another bug here is that the `extension` check was under VentilationRate before, but only HPXML schema only allows `Attic` having `extension` element, so I removed the check under Attic. Let me know your thought.